schema/release-schema.json

```
@@ -549,7 +549,7 @@
         },
         "suppliers": {
           "title": "Suppliers",
-          "description": "The suppliers awarded this award. If different suppliers have been awarded different items or values, these should be split into separate award blocks.",
+          "description": "The suppliers awarded this award. If different suppliers have been awarded different items or values, these should be split into separate award blocks. There can be multiple suppliers for a single award if, for example, the award recipient is an unincorporated consortium or if the award represents a framework agreement set up.",
```

Aligning with the language in the frameworks example:
```suggestion
          "description": "The suppliers awarded this award. If different suppliers have been awarded different items or values, these should be split into separate award blocks. There can be multiple suppliers for a single award if, for example, the award recipient is an unincorporated consortium or if the award represents the addition of suppliers as parties to a framework agreement.",
```